tools/virtualization/setting-up-fedora-workstation-vm-on-qemu-using-uefi.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Setting up Fedora Workstation VM on QEMU using UEFI  
```

Comment:
This could probably be one sub-page `Setting UP Fedora VM...` with various hyperlinks on the top, linking to `UEFI / BIOS / virt-install` headers respectively.

Reply:
That would be two subpages (one for "Setting up Fedora Workstation VM" and one for "QEMU/KVM usages") that would lead to the same place. Would we really want to do that?